doc/signing-at-rksv/rksv-sign-api.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: /poscreators/signing/austria/api
+title: RKSV.Sign API
+---
+
+At it's core, the _RKSV.Sign_ API uses the [IATSSCD](https://github.com/fiskaltrust/middleware-interface-dotnet/blob/master/src/fiskaltrust.ifPOS/v0/at/IATSSCD.cs) interface that is also internally used by the Middleware to communicate with Austrian signing devices and services.
+
+The API is hosted with REST and WCF/SOAP endpoints, users can decide which one to use.
+
+## REST API
+
+### Endpoints
+The REST API is hosted at the following endpoints:
+- **Sandbox**: http://rksv-sandbox.fiskaltrust.at/
+- **Production**: http://rksv.fiskaltrust.at/
+
+:::info
+
+Both of these endpoints offer a fully featured API documentation via Swagger, which also allows automatic client generation. The Swagger documentation is available at the following endpoints: 
+
+- **Sandbox**: https://rksv-sandbox.fiskaltrust.at/swagger/index.html
+- **Production**: https://rksv.fiskaltrust.at/swagger/index.html
+
+For C# users, we've also prepared a sample implementation of the API in the [RKSV.Sign demo repository](https://github.com/fiskaltrust/signing-demo-rksv) on GitHub.
+
+:::
+
+### Authentication
+Like all our services, users of the RKSV.Sign API need to authenticate via _CashboxID_ and _Access Token_, which can be obtained via the Portal. These values need to be included into each requests in the HTTP headers:
+- `cashboxid`: Unique ID of the cashbox, in GUID format
+- `accesstoken`: Secret token that is used to authenticate the request
+
+Additionally to obtaining these values from the Portal, they're also returned once when creating the cashbox via our API.
+
+### Methods
+The following methods are available:
+
+#### Echo
+`POST /api/v0/iatsscd/echo`\
```

Comment:
In the swagger ui the urls are listed as `/v0/iatsscd/echo` and `/api/echo` not `/api/v0/iatsscd/echo` 🤔 are all three correct?

Reply:
Nope, the API is not yet "finalized" 😄 
@StefanKert is working on that, afaik